### ISSUE TITLE
Word change for the Create Screen static scene summary. 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#188]
One word needs to be changed for the static scene summary 

**Was**: 
On this screen you can create your own ratio challenge. The Play Area has two movable hands that can be moved individually or simultaneously, or as a single interaction with the ratio locked to explore the ratio challenges you create. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. In addition, you can increase the range of tick marks for a finer exploration of your ratio challenges. The Control Area has a button to reset the sim.

**Should be changed to:**
On this screen you can **_set_** your own ratio challenge. The Play Area has two movable hands that can be moved individually or simultaneously, or as a single interaction with the ratio locked to explore the ratio challenges you create. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. In addition, you can increase the range of tick marks for a finer exploration of your ratio challenges. The Control Area has a button to reset the sim.

**Comment by zepumph** [comment]
Implemented above. If you want to, you can feel free to edit these minor changes via github directly, see https://github.com/phetsims/ratio-and-proportion/blob/master/ratio-and-proportion-strings_en.json. You can edit that file and link the commit message to the url of an issue like this if you want someone to review it.

**Comment by Matthew-Moore240** [comment]
Looks good! Closing.
